jaeger-core/src/main/java/com/uber/jaeger/reporters/protocols/JaegerThriftSpanConverter.java

```diff
@@ -48,7 +48,7 @@ private JaegerThriftSpanConverter() {}
             context.getTraceId(),
             0, // TraceIdHigh is currently not supported
             context.getSpanId(),
-            context.getParentId(),
+            span.getReferences() == null ? 0 : span.context().getParentId(),
```

Comment:
> shouldn't this be simply context.getParentId() == 0 ? null : context.getParentId()?

`parentId` in thrift is long not an object. I'm proposing to set it to 0 when there are multiple parents.